System/Random.hs

```diff
@@ -665,7 +638,6 @@ programmer to extract random values of a variety of types.
 Minimal complete definition: 'randomR' and 'random'.
 
 -}
-{-# DEPRECATED randomR "In favor of `uniformR`" #-}
```

Comment:
Why do you think we should un-deprecate `randomR`?

Reply:
I removed it for now, because it results in too many warnings everywhere. It looks like it is a very popular function. I think we should wait until next iteration on deprecating it.